ui/src/pages/stop-pending/stop-pending.test.tsx

```diff
@@ -0,0 +1,52 @@
+// src/pages/stop-pending/StopPending.test.tsx
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import '@testing-library/jest-dom';
+import { render, screen } from '@testing-library/react';
+import { BrowserRouter } from 'react-router-dom';
+import { RecoilRoot } from 'recoil';
+import { StopPending } from './stop-pending';
+
+const queryClient = new QueryClient({
+  defaultOptions: {
+    queries: {
+      retry: false,
+    },
+  },
+});
+
+const componentWrapper = (
+  <RecoilRoot>
+    <QueryClientProvider client={queryClient}>
+      <BrowserRouter>
+        <StopPending />
+      </BrowserRouter>
+    </QueryClientProvider>
+  </RecoilRoot>
+);
+
+describe('StopPending', () => {
+  test('should render successfully', async () => {
+    const { baseElement } = render(componentWrapper);
+    expect(baseElement).toBeTruthy();
+    expect(
+      screen.getByText(/Thank you for your patience/i),
+    ).toBeInTheDocument();
+    expect(
+      screen.getByText(
+        /We are stopping your application, you may start it again when we have finished/i,
+      ),
+    ).toBeInTheDocument();
+    expect(screen.getByRole('progressbar')).toBeInTheDocument();
+    expect(
+      screen.getByText(/You may return to the Application Screen at any time/i),
+    ).toBeInTheDocument();
+    expect(
+      screen.getByRole('button', { name: /Back To Home/i }),
+    ).toBeInTheDocument();
+  });
+
+  test('matches snapshot', () => {
+    const { asFragment } = render(componentWrapper);
+    expect(asFragment()).toMatchSnapshot();
+  });
```

Comment:
I don't think we should be using snapshots. Unless you have a strong feeling, lets remove this and the snapshot

Reply:
Removed snapshot